test/ipcFixtures.hpp

```diff
@@ -523,11 +524,11 @@ TEST_P(umfIpcTest, ConcurrentOpenCloseHandles) {
     }
 
     pool.reset(nullptr);
-    EXPECT_EQ(stat.getCount, stat.allocCount);
-    EXPECT_EQ(stat.putCount, stat.getCount);
     // TODO: enale check below once cache for open IPC handles is implemented
+    // EXPECT_EQ(stat.getCount, stat.allocCount);
+    // EXPECT_EQ(stat.putCount, stat.getCount);
     // EXPECT_EQ(stat.openCount, stat.allocCount);
-    EXPECT_EQ(stat.openCount, stat.closeCount);
+    // EXPECT_EQ(stat.openCount, stat.closeCount);
```

Comment:
Does these macros fail because of the new set of tests or they were failing before?

Reply:
Code changed. Removed.
Done.